release.py | Fix handling of files that are not in subdirs, and other minor changes.

aaether32323 opened the pull request:
# Category

This change is exactly one of the following (please change `[ ]` to `[x]`) to indicate which:
* [x] a bug fix (Fix #...)
* [ ] a new Ripper
* [ ] a refactoring
* [ ] a style change/fix


# Description


Fix handling of files that are not in sub-dirs.
Use raw strings for regex, won't change anything as `\d` is not an escape sequence. However it is still good practice.
Should fix #672.

# Testing

Required verification:
* [x] I've verified that this change works as intended.
* [x] I've verified that this change did not break existing functionality (especially in the Ripper I modified).

Optional but recommended:
* [ ] I've added a unit test to cover my change.

cyian-1756 commented:
Testing this on Ubuntu 16.04 with python 3.5.2 produrces an error

```
Traceback (most recent call last):
  File "release.py", line 57, in <module>
    fileToUploadPath = checkAndRenameFile(args.file)
  File "release.py", line 51, in checkAndRenameFile
    os.rename(path, new_path) 
FileNotFoundError: [Errno 2] No such file or directory: 'ripme-1.7.54-jar-with-dependencies.jar' -> 'ripme.jar'
```
When ripme is not in a sub dir (When ripme is in the same dir as release.py)

I invoked the script with the following command

`python3 release.py --file ripme-1.7.54-jar-with-dependencies.jar --token test --test --skip-hash`

aaether32323 commented:
This may sound dumb but are you sure you gave the right argument? Can you post what the directory tree looks like?
This works for me:

```
 > touch ripme-1.7.54-jar-with-dependencies.jar # to create a dummy file
 > python3 release.py --file ripme-1.7.54-jar-with-dependencies.jar --token test --test --skip-hash
Specified file is not named ripme.jar, renaming
[*] WARNING: SKIPPING HASH CHECK
File path: ripme.jar
Release title: 1.7.54: Fixed twitter ripper video downloading; fixed instagram ripper
Repo: ripmeapp/ripme

Please review the information above and ensure it is correct and then press enter
Not uploading release being script was run with --test flag
```

I'm on Python 3.6.5 but I hardly believe that it could make a difference.

cyian-1756 commented:
@aaether32323

It looks like I must of messed up the first time, running it again it works fine
